Webpack incompatibility

Hello there,

It's impossible for me to include `node-github` in a webpack build.

I have this warning (with many others) when I build:

```
WARNING in ./~/github/index.js
Critical dependencies:
779:22-39 the request of a dependency is an expression
 @ ./~/github/index.js 779:22-39
```

And when I run the project, I have:

```
Error: Cannot find module './v3.0.0'.
    at /var/task/index.js:4003:42
    at webpackContextResolve (/var/task/index.js:4003:90)
    at webpackContext (/var/task/index.js:4000:30)
    at new module.exports (/var/task/index.js:263:39)
    at handler (/var/task/index.js:63:19)
```

I saw the closed issue #291, I have the same problem, which is not solved.

--- a/lib/index.js
+++ b/lib/index.js
@@ -777,8 +777,14 @@ var Client = module.exports = function(config) {
         }
 
         function httpSendRequest() {
-            var reqModule = self.config.followRedirects === false ? protocol : 'follow-redirects/' + protocol;
-            var req = require(reqModule).request(options, function(res) {
+            // Verbosity for WebPack compliance
+            if (self.config.followRedirects === false) {
+                var reqModule = protocol == 'http' ? require('http') : require('https')
+            } else {
+                var reqModule = protocol == 'http' ? require('follow-redirects/http') : require('follow-redirects/https')
+            }
+
+            var req = reqModule.request(options, function(res) {
                 if (self.debug) {
                     console.log("STATUS: " + res.statusCode);
                     console.log("HEADERS: " + JSON.stringify(res.headers));

